Added Accrualify section (#574)
mattermost/mattermost-handbook: operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md

## FAQs

**I had an account with Bill.com, do I need to deprovision myself from Bill.com?**

* Nope! The Finance team will deprovision your account for you. You should stop receiving emails from Bill.com by January 31, 2021.

**How do I set up my account on Accrualify?**

> * If you need to submit a PO, review a PO, and/or review an invoice, the Finance team will provision you on Accrualify. You'll receive a Welcome email that will include a link to set up your account. Please note that not all staff members will be provisioned on Accrualify, so if you feel that you need access, please contact Procurement@mattermost.com.

Reviewer: ```suggestion
* If you need to submit a purchase order, review a purchase order, and/or review a vendor invoice, the Finance team will provision you an account on Accrualify. You'll receive a Welcome email that includes a link to set up your account. Please note that not all staff members will be provisioned accounts on Accrualify, so if you feel that you need access, please contact Procurement@mattermost.com.
```

**Where can I find my to-do items on Accrualify?**

> * Once you log into your account, you will be brought to your Dashboard. From there, you will be able to see your to-do items.

Reviewer: ```suggestion
* Once you log into your account, you will be brought to your dashboard. From there, you will be able to see your to-do items.
```

**What are our standard payment terms?**

* We pay vendors 30 days from the date of receipt of the invoice.

**I'm lost and I don’t know where to start, please help!**

> * No worries! Feel free to ping the [Purchases channel](https://community.mattermost.com/private-core/channels/purchases) for any purchasing related questions. The Finance team is here to help and will make sure you feel supported every step of the way!

Reviewer: ```suggestion
* No worries! Feel free to ping the [Purchases channel](https://community.mattermost.com/private-core/channels/purchases) for any purchase-related questions. The Finance team is here to help and will make sure you feel supported every step of the way!
```
